lib/std.dl

```diff
@@ -19,6 +19,12 @@ typedef s128 = signed<128>
 
 typedef usize = u64
 
+typedef DDWeight = s64
+typedef DDEpoch = u64
+typedef DDIteration = u64
+typedef DDTimestamp = DDTSGlobal{epoch: DDEpoch}
```

Comment:
We need to provide a string conversion function for this type so that timestamps can be easily printed. The function must have the following signature: `dDTimestamp2string(ts: DDTimestamp): string`